.circleci/config.yml

```diff
@@ -1091,16 +1091,16 @@ workflows:
             pattern: smoove/0.2.5/Dockerfile
     - docker/publish:
         context: dockerhub-vars
-        name: cellranger_5.0.1_diff
+        name: cellranger_5.0_diff
         deploy: false
         registry: pgc-images.sbgenomics.com
         image: d3b-bixu/cellranger
-        tag: 5.0.1
-        path: cellranger/5.0.1/
-        docker-context: cellranger/5.0.1/
+        tag: '5.0'
```

Comment:
I'm guessing whether the tag is a string or just written as text works either way